src/renderers/index.ts

```diff
@@ -31,7 +32,8 @@ export
 class HTMLWidget extends Widget {
   constructor(html: string) {
     super();
-    this.node.innerHTML = html;
+    let range = document.createRange();
+    this.node.appendChild(range.createContextualFragment(html));
```

Comment:
I can verify the bug, but they appear to have fixed it https://bugs.webkit.org/show_bug.cgi?id=154627.  Since it is working otherwise on the other browsers, I'm in favor of using this solution.